hub/core/augment/transforms.py

```diff
@@ -0,0 +1,286 @@
+import hub
+import cv2 as cv
+import numpy as np
+from torchvision.transforms import TrivialAugmentWide
+import torch
+
+
+def transform_image(comp_func):
+    return comp_func.func(*comp_func.args)
+
+
+def is_grayscale_image(image):
+    """
+    Helper function to spot gray_scale images.
+    """
+    return (len(image.shape) == 2) or (len(image.shape) == 3 and image.shape[-1] == 1)
+
+
+class TrivialAugment(TrivialAugmentWide):
+  def __init__(self,exclude_transforms=None, include_transforms=None):
+    """
+    Inherited class from pytorch TrivialAugmentWide. Helps change Augmentation Space. Can either include
+    transforms or exclude transforms. There are 14 transforms available from the Wide Augmentation Space(refer TA paper)
```

Comment:
mention that this is pytorch only and add examples in the comments so docs can properly render them